### DISCUSSION TITLE
Logging into the mobile builder

### DISCUSSION THREAD

**Discussion by imJohnYeager** [Auties00/WhatsappWeb4j#367] [Q&A]
Can someone please help me? I don't know what to do when it gets to this part. I've read the readme multiple times but can't figure it out on my own. I've spent two sleepless nights on it this weekend... (ignore the "xxxx" on the number, and i'm Brazillian).

![help](https://github.com/Auties00/Cobalt/assets/130397626/71de4593-38c5-4d0a-8d47-e9a6976a60e3)

**Comment by Auties00** [comment]
> Can someone please help me? I don't know what to do when it gets to this part. I've read the readme multiple times but can't figure it out on my own. I've spent two sleepless nights on it this weekend... (ignore the "xxxx" on the number, and i'm Brazillian).
> 
> ![help](https://github.com/Auties00/Cobalt/assets/130397626/71de4593-38c5-4d0a-8d47-e9a6976a60e3)
> 

Just provide a lambda to get the verification code, for example:
() -> {
   System.out.println("Enter code");
   return new Scanner(System.in).nextLine();
}

**Reply by imJohnYeager** [reply]
GOD!!!! TY!
